http/1.6/modules/ROOT/pages/http-connector-xml-maven.adoc

```diff
@@ -0,0 +1,77 @@
+= HTTP Connector 1.6 - XML and Maven Support - Mule 4
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio:
+
+* If you manually code a Mule runtime engine (Mule) app in XML (either from the Anypoint Studio XML editor or from a text editor), you can access the connector from your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
+* If you add the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location and it also adds a dependency to the `pom.xml` file.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `mule` tag of the header
+of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/connector-template-files
```

Comment:
Replace connector-template-files with your connector ID for lines 15 and 16 (probably http-connector)?